src/components/Sections/SectionTable.less

```diff
@@ -2,18 +2,15 @@
   overflow-x: hidden;
   margin-top: 0 !important;
 
-  thead {
-    th {
-      overflow: hidden;
-      text-overflow: ellipsis;
-    }
-  }
-
```

Comment:
please revert this change we want to show the table headers

Reply:
I restored the code the hides the table headers:
```
          table {
            display: none;
          }
```